packages/create-gatsby/src/index.ts

```diff
@@ -297,24 +300,22 @@ ${c.bold(`Thanks! Here's what we'll now do:`)}
 
   await initStarter(DEFAULT_STARTER, data.project, packages.map(removeKey))
 
-  console.log(
-    c.green(c.symbols.check) + ` Created site in ` + c.green(data.project)
-  )
+  reporter.success(`Created site in ` + c.green(data.project))
 
   if (plugins.length) {
-    console.log(c.bold(`${w(`🔌 `)}Installing plugins...`))
+    console.log(`${w(`🔌 `)}Setting-up plugins...`)
```

Comment:
Any reason why you don't use reporter here?

Reply:
Reporter isn't the real `reporter` from the cli. I wrote it as a polyfill when I moved the other code over from gatsby-cli, but kept it because I'd added some useful helpers for `success`, `panic` etc. I didn't bother changing the .info calls, but probably should for consistency.

Reply:
Changed the rest for consistency

Reply:
I was aware of not being the real reporter :) Thanks!